app.js

```diff
@@ -0,0 +1,186 @@
+const { MongoClient } = require("mongodb");
+
+async function run() {
+  // TODO:
+  // Replace the placeholder connection string below with your
+  // Altas cluster specifics. Be sure it includes
+  // a valid username and password! Note that in a production environment,
+  // you do not want to store your password in plain-text here.
+  const uri =
+    "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+
+  // The MongoClient is the object that references the connection to our
+  // datastore (Atlas, for example)
+  const client = new MongoClient(uri);
+
+  // The connect() method does not attempt a connection; instead it instructs
+  // the driver to connect using the settings provided when a connection
+  // is required.
+  await client.connect();
+
+  // Provide the name of the database and collection you want to use.
+  // If the database and/or collection do not exist, the driver and Atlas
+  // will create them automatically when you first write data.
+  const dbName = "myDatabase";
+  const collectionName = "recipes";
+
+  // Create references to the database and collection in order to run
+  // operations on them.
+  const database = client.db(dbName);
+  const collection = database.collection(collectionName);
+
+  /*
+   *  *** INSERT DOCUMENTS ***
+   *
+   * You can insert individual documents using collection.insert().
+   * In this example, we're going to create four documents and then
+   * insert them all in one call with collection.insertMany().
+   */
+
+  const recipes = [
+    {
+      name: "elotes",
+      ingredients: [
+        "corn",
+        "mayonnaise",
+        "cotija cheese",
+        "sour cream",
+        "lime",
+      ],
+      prepTimeInMinutes: 35,
+    },
+    {
+      name: "loco moco",
+      ingredients: [
+        "ground beef",
+        "butter",
+        "onion",
+        "egg",
+        "bread bun",
+        "mushrooms",
+      ],
+      prepTimeInMinutes: 54,
+    },
+    {
+      name: "patatas bravas",
+      ingredients: [
+        "potato",
+        "tomato",
+        "olive oil",
+        "onion",
+        "garlic",
+        "paprika",
+      ],
+      prepTimeInMinutes: 80,
+    },
+    {
+      name: "fried rice",
+      ingredients: [
+        "rice",
+        "soy sauce",
+        "egg",
+        "onion",
+        "pea",
+        "carrot",
+        "sesame oil",
+      ],
+      prepTimeInMinutes: 40,
+    },
+  ];
```

Comment:
Torn on this one: this giant block of JSON really breaks up the code, so it might be a good idea to refactor it into a "getRecipes" function so we don't disrupt the flow when you're trying to read through things for the first time. But I don't think there's anything explicitly wrong with this, so up to you.

Reply:
Breaking up the flow is a valid point.

Personally, I think given the current structure of our code examples, placing it inline maintains the chronologically-ordered pattern of the lesson.  Similar to writing a book, you probably want to establish they know what's going on as they progress through the first pages rather than asking them to navigate to another part of the book to get that information. 

My understanding of why we picked this chronologically ordered was specifically for that. Otherwise, there are other ways to structure the code that could be more appropriate such as this modular approach:
https://github.com/ccho-mongodb/atlas-example-node/blob/master/example.js

While I do not have studies proving the efficacy of either direction, I think this is a case in which a relatively small amount of sample data is better to surface in the order that it is needed without adjusting the pattern of navigation. If we are teaching people best practices for structuring their app or how to read a codebase, we should restructure our model to reflect that.